cms/static/js/views/active_video_upload_list.js

```diff
@@ -307,6 +373,16 @@ define([
                     completedMessages.push(gettext('Previous Uploads table has been updated.'));
                     this.readMessages(completedMessages);
                 }
+            },
+
+            sendStatusUpdate: function(statusUpdates) {
+                return $.ajax({
+                    url: this.postUrl,
+                    contentType: 'application/json',
+                    data: JSON.stringify(statusUpdates),
+                    dataType: 'json',
```

Comment:
We are sending empty json response from server so i think we should keep `dataType`. We can discuss if you think we should remove it.